crates/pop-common/src/git.rs

```diff
@@ -503,6 +549,37 @@ mod tests {
 	fn parse_latest_tag_works() {
 		let mut tags = vec![];
 		assert_eq!(Git::parse_latest_tag(tags), None);
+		tags = vec![
+			"polkadot-v1.10.0",
+			"polkadot-v1.11.0",
+			"polkadot-v1.12.0",
+			"polkadot-v1.7.0",
+			"polkadot-v1.8.0",
+			"polkadot-v1.9.0",
+			"polkadot-stable2407",
```

Comment:
This list isnt sorted alphabetically, which is the order that GitRepository seems to return tags in. Best to double check tho.